spec/lib/html_cleaner_spec.rb

```diff
@@ -6,11 +6,11 @@
 
   describe "sanitize_value" do
     ArchiveConfig.FIELDS_ALLOWING_VIDEO_EMBEDS.each do |field|
-      context "#{field} is configured to allow video embeds" do
+      context "#{field} is configured to allow media embeds" do
```

Comment:
Reworded to "media" since this includes more than just videos, though I guess this would suggest refactoring `FIELDS_ALLOWING_VIDEO_EMBEDS` too

Reply:
🫡 Done! [d8c62d3](https://github.com/otwcode/otwarchive/pull/5203/commits/d8c62d3ccf1360c637f17629fb64c7d67bc74e9d)